components/blitz/src/ome/formats/importer/cli/CommandLineImporter.java

```diff
@@ -492,19 +488,31 @@ public static void advUsage() {
     private static List<Annotation> toTextAnnotations(
                    List<String> namespaces, List<String> strings)
     {
-        if (namespaces.size() != strings.size())
-        {
-            throw new IllegalArgumentException(String.format(
-                            "#Namespaces:%d != #Text:%d", namespaces.size(),
-                            strings.size()));
-        }
         List<Annotation> annotations = new ArrayList<Annotation>();
-        for(int i = 0; i < namespaces.size(); i++)
+        CommentAnnotationI annotation;
+        if (namespaces.size() != strings.size())
         {
-            CommentAnnotationI annotation = new CommentAnnotationI();
-            annotation.setNs(omero.rtypes.rstring(namespaces.get(i)));
-            annotation.setTextValue(omero.rtypes.rstring(strings.get(i)));
-            annotations.add(annotation);
+            String ns = null;
+            if (namespaces.size() == 1) {
+                ns = namespaces.get(0);
+            }
+            for(int i = 0; i < strings.size(); i++)
+            {
+                annotation = new CommentAnnotationI();
+                if (ns != null) {
+                    annotation.setTextValue(omero.rtypes.rstring(ns));
+                }
+                annotation.setTextValue(omero.rtypes.rstring(strings.get(i)));
+                annotations.add(annotation);
+            }
+        } else {
```

Comment:
`./bin/omero import --annotation-ns testNameSpace --annotation-text testAnnotation /Users/bramalingam/Desktop/Screen\ Shot\ 2017-05-09\ at\ 11.57.33.png `

this works without any issues